src/framework/components/model/component.js

```diff
@@ -19,15 +19,39 @@ import { Component } from '../component.js';
  */
 
 /**
- * Enables an Entity to render a model or a primitive shape. This Component attaches additional
- * model geometry in to the scene graph below the Entity.
+ * The ModelComponent enables an {@link Entity} to render 3D models. The {@link type} property can
+ * be set to one of several predefined shapes (such as `box`, `sphere`, `cone` and so on).
+ * Alternatively, the component can be configured to manage an arbitrary {@link Model}. This can
+ * either be created programmatically or loaded from an {@link Asset}.
+ *
+ * The {@link Model} managed by this component is positioned, rotated, and scaled in world space by
+ * the world transformation matrix of the owner {@link Entity}. This world matrix is derived by
+ * combining the entity's local transformation (position, rotation, and scale) with the world
+ * transformation matrix of its parent entity in the scene hierarchy.
```

Comment:
it seems that most of this should be in GraphNode?

Reply:
I have update GraphNode class overview too.

Reply:
yep really good, even I'm starting to understand it now.